static/app/components/forms/controls/selectControl.tsx

```diff
@@ -269,7 +269,11 @@ function SelectControl<OptionType extends GeneralSelectValue = GeneralSelectValu
       menuList: provided => ({
         ...provided,
         ...(isCompact && {
-          paddingTop: isSearchable ? 0 : `calc(${space(0.5)} + 1px)`,
+          paddingTop: isSearchable
+            ? 0
+            : menuTitle
+            ? `calc(${space(0.5)} + 1px)`
+            : space(0.5),
```

Comment:
Can we get a comment as to why the 1px is needed here.

in the future if we come across this it would be nice to know why, so we don't have to re-figure that out haha